charts/umbrella/README.md

```diff
@@ -1,910 +1,45 @@
-```
-
-```
-tractusxpgdamin4
-```
-
-**The database server connections need to be added manually to pgadmin4.**
-
-Default username for all connections:
-
-```
-postgres
-```
-
-Default user for all connections:
-
-```
-5432
-```
-
-In the following some of the available connections:
-
-- portal
-
-Host:
-
-```
-umbrella-portal-backend-postgresql
-```
-
-Password:
-
-```
-dbpasswordportal
-```
-
-- centralidp
-
-Host:
-
-```
-umbrella-centralidp-postgresql
-```
-
-Password:
-
-```
-dbpasswordcentralidp
-```
-
-- sharedidp
-
-Host:
-
-```
-umbrella-sharedidp-postgresql
-```
-
-Password:
-
-```
-dbpasswordsharedidp
-```
-
-- miw
-
-Host:
-
-```
-umbrella-miw-postgres
-```
-
-Password:
-
-```
-dbpasswordmiw
-```
-
-- dataprovider
-
-Host:
-
-```
-umbrella-dataprovider-db
-```
-
-Password:
-
-```
-dbpasswordtxdataprovider
-```
-
-- dataconsumer-1
-
-Host:
-
-```
-umbrella-dataconsumer-1-db
-```
-
-Password:
-
-```
-dbpassworddataconsumerone
-```
-
-- dataconsumer-2
-
-Host:
-
-```
-umbrella-dataconsumer-2-db
-```
-
-Password:
-
-```
-dbpassworddataconsumertwo
-```
-
-- bpdm
-
-Host:
-
-```
-umbrella-bpdm-postgres
-```
-
-Password:
-
-```
-dbpasswordbpdm
-```
-
-### Keycloak Admin Console
-
-Access to admin consoles:
-
-- http://centralidp.tx.test/auth/
-- http://sharedidp.tx.test/auth/
-
-Default username for centralidp and sharedidp:
-
-```
-admin
-```
-
-Password centralidp:
-
-```
-adminconsolepwcentralidp
-```
-
-Password sharedidp:
-
-```
-adminconsolepwsharedidp
-```
-
-
-### Seeding
-
-See [Overall Seeding](../../concept/seeds-overall-data.md).
-
-### Self-signed TLS setup (Optional)
-
-Some of the components are prepared to be configured with TLS enabled (see "uncomment the following line for tls" comments in [values.yaml](./values.yaml)).
-
-If you'd like to make use of that, make sure to to execute this step beforehand.
-
-Install cert-manager chart in the same namespace where the umbrella chart will be located.
-
-```bash
-helm repo add jetstack https://charts.jetstack.io
-helm repo update
-```
-
-```bash
-helm install \
-  cert-manager jetstack/cert-manager \
-  --namespace umbrella \
-  --create-namespace \
-  --version v1.14.4 \
-  --set installCRDs=true
-```
-
-Configure the self-signed certificate and issuer to be used by the ingress resources.
-
-If you have the repository checked out you can run:
-
-```bash
-kubectl apply -f ./charts/umbrella/cluster-issuer.yaml
-```
-
-or otherwise you can run:
-
-```bash
-kubectl apply -f - <<EOF
-apiVersion: cert-manager.io/v1
-kind: ClusterIssuer
-metadata:
-  name: selfsigned-issuer
-spec:
-  selfSigned: {}
+> The versions of integrated components correspond to the **overarching [Release 24.05](https://github.com/eclipse-tractusx/tractus-x-release/blob/24.05/CHANGELOG.md#2405---2024-05-29)**. The upgrade to **[Release 24.08](https://github.com/eclipse-tractusx/tractus-x-release/blob/24.08/CHANGELOG.md#2408---2024-08-05)** is currently in progress, some components have already been upgraded.
```

Comment:
upgrade has been completed by now.
```suggestion
> The versions of integrated components correspond to the **overarching [Release 24.08](https://github.com/eclipse-tractusx/tractus-x-release/blob/24.08/CHANGELOG.md#2408---2024-08-05)** .
```
You could also move this information to the readme on root level.